trifid/redirect.js

```diff
@@ -54,7 +55,19 @@ function factory () {
         }
       }
       if (shapePath) {
-        return res.redirect(`https://raw.githubusercontent.com/zazuko/cube-link/${versionPath}/validation/${shapePath}.ttl`)
+        return fetch(`https://raw.githubusercdontent.com/zazuko/cube-link/${versionPath}/validation/${shapePath}.ttl`)
+          .then(rawGithub => {
+            if (rawGithub.ok) {
+              res.set('Content-Type', 'text/turtle')
+            } else {
+              res.status(500)
+            }
+            Readable.fromWeb(rawGithub.body).pipe(res)
+          })
+          .catch((e) => {
+            res.status(502)
+            res.send(`Error fetching shape: ${e.message}`)
+          })
```

Comment:
We can avoid using `then().catch()` syntax as we are already in an async function.

I also needed to add a type assertion for the body, as else it was complaining. The body can be `null`, therefore I added an explicit check.

Also, with this suggestion, when a shape doesn't exist, it returns a 404 status code instead of a 500.

```suggestion
        try {
          const rawGithub = await fetch(`https://raw.githubusercontent.com/zazuko/cube-link/${versionPath}/validation/${shapePath}.ttl`)
          if (rawGithub.ok) {
            res.set('Content-Type', 'text/turtle')
          } else {
            res.status(500)
          }

          // if the shape does not exist, we return a 404
          if (rawGithub.status === 404) {
            return res.sendStatus(404)
          }

          /** @type {any | null} */
          const body = rawGithub.body
          if (!rawGithub.body) {
            throw new Error('No body')
          }
          return Readable.fromWeb(body).pipe(res)
        } catch (e) {
          return res.status(502).send(`Error fetching shape: ${e.message}`)
        }
```